src/ai/backend/storage/weka/exceptions.py

```diff
@@ -0,0 +1,39 @@
+class WekaError(Exception):
+    pass
+
+
+class WekaInitError(WekaError):
+    pass
+
+
+class WekaAPIError(WekaError):
+    name = 'WekaAPIError'
+    message = ''
+
+    def __init__(self, message: str = None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
It should transparently pass the `message` argument as the part of `args` passed to the super consturctor (See also: [`args` of `BaseException`](https://docs.python.org/3/library/exceptions.html#BaseException.args), [PEP-352](https://peps.python.org/pep-0352/)).
[Stripping arguments for the super constructor affects *picklability* of exceptions.](https://stackoverflow.com/questions/16244923/how-to-make-a-custom-exception-class-with-multiple-init-args-pickleable)  If exceptions are not picklable, it will make it hard to read the stderr exception logs when the service uses `ai.backend.common.logging` with a multi-process architecture.

```python
class WekaAPIError(WekaError):
    message: str

    def __init__(self, message: Optional[str] = None, *args):
        # Unless we explicitly utilize keyword arguments,
        # there is no need to pass them to the super constructor.
        super().__init__(message, *args)
        # just additionally define typed attributes
        self.message = message or ""
```